core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/ClusterDbCertRotationRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.sequenceiq.cloudbreak.reactor.api.event.cluster;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.cloudbreak.reactor.api.ClusterPlatformRequest;
+
+public class ClusterDbCertRotationRequest extends ClusterPlatformRequest {
```

Comment:
Why not subclass `StackEvent` instead?

Reply:
The entire flow is mixed, I don't know why it works. I have followed the flow pattern and I have accepted that fact it is working

Reply:
However with more time I can dug into the code and try to figure out what happen here

Reply:
Let us leave it as is for now; can be investigated & refactored later if needed.